C:\Users\nicka\PycharmProjects\hadoop_hw2\venv\Scripts\python.exe "C:\Program Files\JetBrains\PyCharm Community Edition 2017.3.4\helpers\pycharm\_jb_unittest_runner.py" --path C:/Users/nicka/PycharmProjects/hadoop_hw2/src/tests/test_for_reducer.py
Launching unittests with arguments python -m unittest C:/Users/nicka/PycharmProjects/hadoop_hw2/src/tests/test_for_reducer.py in C:\Users\nicka\PycharmProjects\hadoop_hw2\src\tests
configure_options() is deprecated and will be removed in v0.7.0; please use configure_args() instead.
add_passthrough_option() is deprecated and will be removed in v0.7.0. Use add_passthru_arg() instead.


Ran 2 tests in 0.010s

OK
configure_options() is deprecated and will be removed in v0.7.0; please use configure_args() instead.
add_passthrough_option() is deprecated and will be removed in v0.7.0. Use add_passthru_arg() instead.
